Spanner/tests/System/PgBatchWriteTest.php

```diff
@@ -28,6 +28,7 @@ class PgBatchWriteTest extends SpannerPgTestCase
     const TABLE_NAME = 'BatchWrites';
     public static function setUpBeforeClass(): void
     {
+        // Also skipped for GSQL.
```

Comment:
What is this in reference to? Are you assuming the developer looked at `BgBatchTest` first?

Let's remove "Also" here. Something like this (if it is correct):
```suggestion
        // Emulator does not support GSQL for PG.
```